### PR TITLE
Dịch hình chương 13 - Phần 1

### LINKED ISSUE #2245: Dịch hình chương 13 - Phần 1
Các hình cần dịch:
* [finetune.svg](https://github.com/aivivn/d2l-vn/blob/master/img/finetune.svg)
* [iou.svg](https://github.com/aivivn/d2l-vn/blob/master/img/iou.svg)
* [anchor-label.svg](https://github.com/aivivn/d2l-vn/blob/master/img/anchor-label.svg)
* [ssd.svg](https://github.com/aivivn/d2l-vn/blob/master/img/ssd.svg)

Xem hướng dẫn dịch hình ảnh **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

### PR DESCRIPTION
**Pull request by cuongvng**
Close #2245
@rootonchair review giúp mình nhé. Còn chưa thống nhất `pre-train` thôi, cần thì sửa lại cũng dễ.

### CODE REVIEW

**Comment by rootonchair** [comment]
Mình thấy ổn, riêng `pre-train` cứ để vậy. Sau này nếu có gì mình sẽ sửa
